src/core/geometry/qgsgeometry.h

```diff
@@ -1625,6 +1625,16 @@ class CORE_EXPORT QgsGeometry
      */
     QgsGeometry convexHull() const;
 
+    /**
+     * Returns a posibbly concave polygon that contains all the points in the geometry.
```

Comment:
```suggestion
     * Returns a posibly concave polygon that contains all the points in the geometry.
```

But why "posibly" anyway?

This also needs the sip_raises and \throws doxygen addition

Reply:
>  But why "posibly" anyway?

It's the postgis wording. I guess since it's not a definitive concave polygon, since there are multiple algorithms to return a concave polygon that are not equal.